remake.yml

```diff
@@ -119,7 +119,8 @@ targets:
       sb_id = sbid_04_predictions,
       sources = 'src/sb_utils.R',
       'out_data/reservoir_io_sntemp.csv',
-      'out_data/reservoir_downstream_preds.csv')
+      'out_data/reservoir_downstream_preds.csv',
+      "in_data/dwallin_stream_preds.csv")
```

Comment:
Why not run a file.copy() into out_data for dwallin_stream_preds.csv? Like it is done for reservoir_downstream_preds.csv.  

I am leaning towards prefering that dwallin_stream_preds.csv gets copied and ziped since it is quite large. Do you agree?

Reply:
see changes from commit https://github.com/USGS-R/forecast-data-release-v2/pull/5/commits/5e4925fd0e8bbc0da5c32cca52960236c3c7fe81

Reply:
The reason I wasn't doing `file.copy` was because I was pushing the file as-is, so could point directly to the file. Yeah, it's fine to zip and push. I'll ping Jake when it's changed since they're pointing to this file.